src/subscription_manager/cli_command/abstract_syspurpose.py

```diff
@@ -155,6 +155,16 @@ def _validate_options(self):
             else:
                 system_exit(ERR_NOT_REGISTERED_CODE, ERR_NOT_REGISTERED_MSG)
 
+        if self.is_registered() and (
+            getattr(self.options, "username", None) or
+            getattr(self.options, "password", None) or
+            getattr(self.options, "token", None)
+        ):
+            system_exit(os.EX_USAGE, _(
+                "Error: --username, --password and --token "
+                "can only be used on unregistered system"
```

Comment:
Linguistic question: Wouldn't be better to use "Error: --username, --password and --token can be used only on unregistered system." ? @tlhmerry9 @cnsnyder Thoughts?

Reply:
@jirihnidek as non-English speaker: from what I know, there is a small difference in the meaning of the two versions (the current and yours):
- "it can only be used on ..." == "the ability [can] applies only on the usability of that on ..."
- "it can be used only on ..." == "you can use ... only on"

Again, AFAIK. In the end, I think the general meaning of both should be the same in this context.

Reply:
OK :smiley:

Reply:
I think it is fine as is, with one small change "Error: --username, --password and --token can be used only on an unregistered system."

Reply:
Fixed.